config/etfuturum/functions.cfg

```diff
@@ -0,0 +1,278 @@
+
+    # Make F3 only show/hide info on release, and not if another key is pressed [default: true]
+    B:enableNewF3Behavior=true
+
+    # Replace tall grass and sponge textures with modern version [default: true]
+    B:enableNewTextures=true
+
+    # Allows use of 1.8 skin format, and Alex skins. Also includes some fixes from SkinPort. (Per SkinPort author's permission) Disable if skin is displaying oddly. Not compatible with OptiFine, use FastCraft instead. [default: false]
+    B:enablePlayerSkinOverlay=false
+
+    # Enable subtitles [default: false]
+    B:enableSubtitles=false
+
+    # Allow non-opaque armour [default: true]
+    B:enableTransparentAmour=true
+
+    # Render beds with a 3D inventory model instead of a 2D sprite. [default: true]
+    B:inventoryBedModels=true
+
+    # The maximum amount of items a Shulker box can display on its tooltip. When the box has more stacks inside than this number, the rest of the stacks are displayed as "And x more...". Set to 0 to disable Shulker Box tooltips. [range: 0 ~ 127, default: 5]
+    I:shulkerBoxTooltipLines=5
+
+    # Regex of subtitles to blacklist [default: ^(dig\.*)]
+    S:subtitleBlacklist=^(dig\.*)
+}
+
+
+##########################################################################################################
+# settings
+#--------------------------------------------------------------------------------------------------------#
+# Settings for Et Futurum content.
+##########################################################################################################
+
+settings {
+    # The data watcher flag for the Elytra, used to sync the elytra animation with other players. In vanilla the max value is 7, players use 0-4, so you can set this to 6 or 7 by default. ASJCore increases the max value to 31.
+    # Do not change this value if you don't need to, or do not know what you're doing. [range: 0 ~ 31, default: 7]
+    I:elytraDataWatcherFlag=7
+
+    # Seeks all available smeltable ores, metals, etc (using OreDict tags like "ore", "cluster", "ingot", etc) from the furnace and adds them to the Blast Furnace, if it's off it will only smelt things specified from CraftTweaker. [default: true]
+    B:enableAutoAddBlastFurnace=true
+
+    # Seeks compostable items like plants and leaves, and automatically adds them to the composter. If this is off it will only compost things specified from CraftTweaker and nothing by default. [default: true]
+    B:enableAutoAddComposter=true
+
+    # Seeks all available edible foods from the furnace and adds them to the smoker, if it's off it will only smelt things specified from CraftTweaker. [default: true]
+    B:enableAutoAddSmoker=true
+
+    # Set to true to disable the fireproof item entity Netherite/ancient debris etc uses [default: false]
+    B:enableNetheriteFlammable=false
+
+    # Recipe for prismarine if you want it without the temples, or want it craftable alongside temples. [default: true]
+    B:enablePrismarineRecipes=true
```

Comment:
probably better to make a GTNH specific recipe?

Reply:
It was disabled in the ETF code for GTNH. I'll also set false here to be doubly sure.